src/main/java/core/basesyntax/MyHashMap.java

```diff
@@ -1,19 +1,179 @@
 package core.basesyntax;
 
+import static java.util.Objects.hash;
+
+import java.util.Map;
+import java.util.Objects;
+
 public class MyHashMap<K, V> implements MyMap<K, V> {
+    public static final int DEFAULT_INITIAL_CAPACITY = 16;
+    public static final float DEFAULT_LOAD_FACTOR = 0.75f;
+    private int size;
+    private int threshold;
+    private Node<K, V>[] table = new Node[DEFAULT_INITIAL_CAPACITY];
+
+    static class Node<K, V> {
+        private final int hash;
+        private final K key;
+        private V value;
+        private Node<K, V> next;
+
+        Node(int hash, K key, V value, Node<K, V> next) {
+            this.hash = hash;
```

Comment:
This field makes code less readable and has no real purpose.
Index in table can be calculated from key hashcode
```suggestion
```